en/installation/installation-of-a-poller/using-packages.md

```diff
@@ -68,6 +83,106 @@ systemctl start centreontrapd snmptrapd
 > Active monitoring service will be started following the generation of its
 > configuration.
 
+## Register the server
+
+To register it to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <REMOTE_SERVER_NAME>
```

Comment:
-n <REMOTE_SERVER_NAME>

here we are registering a Poller, so this is a Poller name ?